src/Mockman.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
```

Comment:
No need to import React in individual components, it will work without it.

Reply:
okay will check